apps/subclass_ndarray/include/subndarray.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TVM_SUBNDARRAY_H_
+#define TVM_SUBNDARRAY_H_
+
+#include <tvm/runtime/packed_func.h>
```

Comment:
Please move this test case to to apps/extension

Reply:
ok